js-objects-and-arrays_objects/index.js

```diff
@@ -26,6 +30,11 @@ const personToChange = {
   age: 24,
 };
 
+personToChange.name = "Ivan";
```

Comment:
you changed it to the wrong name but I don't think it really matters here :)

Reply:
I had to put Alex, not Ivan I got it. Thank you!